x/leveragelp/keeper/position.go

```diff
@@ -312,6 +242,8 @@ func (k Keeper) GetPositions(ctx sdk.Context, pagination *query.PageRequest) ([]
 		var position types.Position
 		err := k.cdc.Unmarshal(value, &position)
 		if err == nil {
+			debt := k.stableKeeper.GetDebt(ctx, position.GetPositionAddress())
+			position.Liabilities = k.stableKeeper.GetInterest(ctx, debt.LastInterestCalcBlock, debt.LastInterestCalcTime, debt.Borrowed.ToLegacyDec())
```

Comment:
Why can't we fetch the liabilities separately wherever its required. this will slow down queries where it's not needed

Reply:
it was required from off-chain bot, we may put it separate but then two queries will be needed, This is fine and more accurate